README.md

```diff
@@ -23,22 +23,27 @@
 
 <div align="center">
 
-| **classification_one_inch_maps_001**<br><a href="https://github.com/Living-with-machines/MapReader/tree/main/examples/classification_one_inch_maps_001"> <img src="figs/tutorial_classification_one_inch_maps_001.png" alt="tutorial for classification_one_inch_maps_001" style="height:200px;"></a><br><sup>Tutorial on how to train/fine-tune PyTorch CV classifiers on historical maps. Figure shows the rail infrastructure around London as predicted by a MapReader CV model.</sup> | **classification_plant_phenotype**<br><a href="https://github.com/Living-with-machines/MapReader/tree/main/examples/classification_plant_phenotype"> <img src="figs/tutorial_classification_plant_phenotype.png" alt="tutorial for classification_plant_phenotype" style="height:200px;"></a><br><sup>Tutorial on how to train/fine-tune PyTorch CV classifiers on plant patches in images (plant phenotyping example).</sup> |
+| **classification_one_inch_maps_001**<br><a href="https://github.com/Living-with-machines/MapReader/tree/main/examples/maps/classification_one_inch_maps_001"><img src="figs/tutorial_classification_one_inch_maps_001.png" alt="tutorial for classification_one_inch_maps_001" style="height:200px;"></a><br><sup>Tutorial: train/fine-tune PyTorch CV classifiers on <ins>historical maps</ins>. Fig shows the rail infrastructure around London as predicted by a MapReader CV model.</sup> | **classification_plant_phenotype**<br><a href="https://github.com/Living-with-machines/MapReader/tree/main/examples/non-maps/classification_plant_phenotype"><img src="figs/tutorial_classification_plant_phenotype.png" alt="tutorial for classification_plant_phenotype" style="height:200px;"></a><br><sup>Tutorial: train/fine-tune PyTorch CV classifiers on <ins>plant patches</ins> in images (plant phenotyping example).</sup> |
 |:---:|:---:|
-| **MapReader paper**<br><a href="https://arxiv.org/abs/2111.15592"> <img src="figs/mapreader_paper.png" alt="MapReader's paper" style="height:200px;"> </a>  |  |
-
+| **classification_mnist**<br><a href="https://github.com/Living-with-machines/MapReader/tree/main/examples/non-maps/classification_mnist"><img src="figs/tutorial_classification_mnist.png" alt="tutorial for classification_mnist" style="height:130px;"></a><br><sup>Tutorial: train/fine-tune PyTorch CV classifiers on <ins>MNIST</ins>.</sup> | |
 </div>
 
-## What is MapReader?
+# What is MapReader?
+
+MapReader is an end-to-end computer vision (CV) pipeline for analyzing and exploring large collections of maps and images. 
+
+MapReader was developed in the [Living with Machines](https://livingwithmachines.ac.uk/) project to analyze large collections of historical maps but is a _**generalisable**_ computer vision pipeline which can be applied to _**any images**_ in a wide variety of domains. See [Gallery](#gallery) for some examples.
```

Comment:
@kmcdono2 I think we should keep the `What is MapReader?` section punchy and to the point (i.e., generalizability), but please do move any of those sentences to `MapReader/examples/maps/README.md`.

Reply:
@kasra-hosseini @dcsw2 This is the opening para where we wanted to insert some version of Daniel's comment re: cross-disciplinary work. It doesn't have to be what is written above, it could be just 1 extra sentence if done in a pithy way. I think this is the core content to add: "We were inspired by CV solutions for medical imaging (CITATION?) and research in the growing field of historical map processing (CITE chiang et al book)."

Reply:
@kmcdono2 I think we have talked about this in the paper. Can you explain how this helps MapReader users? This way, I can think of the best place to add that sentence.

Reply:
Before I forget, I think @dcsw2 's sentences will be added after the bullet points. See: https://github.com/Living-with-machines/MapReader/pull/41#issue-1214549018

Reply:
Please add the new sentences today to this PR or, of course, please feel free to open a new PR later.